File: characters.py
class Characters:

    def __init__(self, tank, assasin, wizard, healer, attack_damage, health, defence, heal):
        self.tank = tank
        self.assasin = assasin
        self.wizard = wizard
        self.healer = healer
        self.attack_damage = attack_damage
        self.__health = health
        self.defence = defence
        self.heal = heal

    def Tank(self):
        self.__health = 150
        self.attack_damage = 10
        self.defence = 30
        self.heal = 10

    def Assasin(self):
        self.__health = 80
        self.attack_damage = 25
        self.defence = 10
        self.heal = 10

    def Wizard(self):
        self.__health = 100
        self.attack_damage = 20
        self.defence = 20
        self.heal = 10

    def Healer(self):
        self.__health = 100
        self.attack_damage = 15
        self.defence = 15
        self.heal = 25

    def get(self):
        return self.__health

File: characters.rb
class Characters
    attr_accessor :tank, :assasin, :wizard, :healer, :attack_damage, :defence, :heal

    def initialize(tank, assasin, wizard, healer, attack_damage, health, defence, heal)
      @tank = tank
      @assasin = assasin
      @wizard = wizard
      @healer = healer
      @health = health
      @attack_damage = attack_damage
      @defence = defence
      @heal = heal
    end

    def tank
      @health = 150
      @attack_damage = 10
      @defence = 30
      @heal = 10
    end

    def assasin
      @health = 80
      @attack_damage = 25
      @defence = 10
      @heal = 10
    end

    def wizard
      @health = 100
      @attack_damage = 20
      @defence = 20
      @heal = 10
    end

    def healer
      @health = 100
      @attack_damage = 15
      @defence = 15
      @heal = 25
    end

    def get
      @health
    end
  end
